docs/toc.json

```diff
@@ -17,14 +17,26 @@
 				{"docs/designers-developers/developers/filters/parser-filters.md": []},
 				{"docs/designers-developers/developers/filters/autocomplete-filters.md": []}
 			]},
+			{"docs/designers-developers/developers/internationalization.md": []},
 			{"docs/designers-developers/developers/data/README.md": "{{data}}"},
 			{"docs/designers-developers/developers/packages.md": "{{packages}}"},
+			{"packages/components/README.md": "{{components}}"},
```

Comment:
So one idea here with the packages and data docs and components docs is supporting some kind of wildcard here, e.g.:

```json
{"packages/components/README.md": [
  {"packages/components/*": []}
]},
```

This would allow us to separate the build and the manifest generation, such that there's a distinct build step that generates or syncs the markdown files into the `docs/` folder, and then a separate step for adding them to the `manifest.json`.

(It's a wildcard rather than explicitly listing files here to allow us to have sections of documentation where the files inside the nesting level may change but the documentation's order doesn't matter. Just a way to simplify a bit here.)

Does that make sense as a way to improve here?

Reply:
Coming back to this, the components and packages documentation offer some weird pieces here because their documentation lives outside the docs folder and can be quite structurally different.

I guess you'd need to either support full globbing, e.g. 

```json
{"packages/components/README.md": [
  {"packages/components/**/*": []}
]},
```

OR… we would have an explicit build step that copies the components/packages documentation _into_ the docs folder first, then a simple wildcard could be used that references the documentation as it exists inside docs, rather than referencing it from elsewhere in the repo.

Thinking out loud, at this point :)